scripts/set-workspace-version.js

```diff
@@ -75,10 +75,6 @@ try {
   process.exit(1);
 }
 
-/**
- * @typedef {{path: string; pkg: Package}} Workspace
- */
-
```

Comment:
Bad merge?

Reply:
Ah yeah, probably. Oddly, i did run the `npm run setversion` script. I'll try again and see what missed.

Reply:
Yes, but the package was already at 2.0.1, and the lock file wasn't updated. This problem only exists because the old script changed things partially.

Reply:
Ah right. That's why we set to 2.0.0, makes sense.